src/Sylius/Bundle/CoreBundle/Doctrine/ORM/ProductRepository.php

```diff
@@ -207,6 +207,23 @@ public function findOneByIdAndChannel($id, ChannelInterface $channel = null)
     /**
      * {@inheritdoc}
      */
+    public function findByTaxonCodeAndChannel($code, ChannelInterface $channel)
```

Comment:
We should also consider getting only enabled products.

``findEnabledByTaxonCodeAndChannel``?